src/less/overwrites/_modal.less

```diff
@@ -8,7 +8,7 @@
 //     <div class="modal-content">
 //       <div class="modal-header">
 //         <button type="button" class="close" data-dismiss="modal">&times;</button>
-//         <h4 class="modal-title">Modal Header</h4>
+//         <h1 class="modal-title">Modal Header</h1>
```

Comment:
I'm not sure if this is a good idea, as the modal is part of the normal dom tree of a page. This means we end up with two <h1>'s even if the modal is not visible. Maybe here a special style for the modal header would be more fitting?